trl/trainer/grpo_trainer.py

```diff
@@ -78,7 +78,7 @@
 RewardFunc = Union[str, PreTrainedModel, Callable[[list, list], list[float]]]
 
 
-class RepeatRandomSampler(Sampler):
+class RepeatSampler(Sampler):
```

Comment:
This renaming makes sense. But to ensure backward compatibility, can you just add something like

```python
class RepeatRandomSampler(RepeatSampler):
    def __init__(self, *args, **kwargs):
        warnings.warn(
            "RepeatRandomSampler is deprecated and will be removed in version 0.18. Use RepeatSampler instead",
            DeprecationWarning,
        )
        super().__init__(*args, **kwargs)
```

Reply:
And you also need to rename in `_get_train_sampler`